chat.py

```diff
@@ -109,6 +109,7 @@ def get_join_url():
     joinParams['userdata-bbb_force_listen_only'] = "true" 
     joinParams['userdata-bbb_skip_check_audio'] = 'true' 
     return bbbUB.buildUrl("join", params=joinParams) 
+    joinParams['joinViaHtml5'] = 'true'
```

Comment:
the param must be above the return of the function - otherwise it will be ignored as the function returns before the line. ;-)please move the line up, so it is before the return.
you can contact me via mail at list(AT)mtsonline(dot)at 
cheers